fastlane/Fastfile

```diff
@@ -0,0 +1,25 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Prepare for release"
+  lane :release  do |options|
+    version_bump_podspec(path: "Bow.podspec", version_number: options[:version_number])
+    add_git_tag(tag: options[:version_number], message: (File.read("../CHANGELOG") rescue "No changelog provided"))
+    push_git_tags
```

Comment:
`pod lib lint` is failing at the moment on master. I'll look into it for a fix and then integrate `Propagate to CocoaPod the Bow modules using the podspecs`

Reply:
Which pods are failing? During release of version 0.6.0 I had to remove support for watchOS and tvOS from the Effects module and the ones depending on it, which may be related to this issue. We need to bring back that support, but I can create a separate issue to track it.

Reply:
When I run ‘pod lib lint’ inside the root directory it fails around swift versioning, I haven’t looked much into it, I’ll do that

Reply:
Ah so the issue was I had a version of cocoapods which did not support `swift_versions`. I have added cocoapods version to Gemfile at the root directory to prevent others from facing a similar issue. `pod lib lint` ran successfully.